jumpstart-foundation-models/text2text-generation-flan-t5-xl/text2text-generation-flan-t5-xl.ipynb

```diff
@@ -0,0 +1,1455 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    sentence = "This moive is so great and once again dazzle and delight us"
</pre><p>dazzles.... delights</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3795/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
resolved.